src/vos/vos_dtx.c

```diff
@@ -290,8 +292,8 @@ dtx_obj_rec_exchange(struct umem_instance *umm, struct vos_obj_df *obj,
 		 * record will be removed via VOS aggregation or other
 		 * tools some time later.
 		 */
-		if (abort)
-			dtx_set_aborted(&obj->vo_dtx);
+		D_ASSERT(abort);
```

Comment:
source is misspelled on line 289.

Reply:
I will fix it when refresh the patch next time.